oem/firstvoices/ios/FirstVoices/AppDelegate.swift

```diff
@@ -22,13 +22,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     FVRegionStorage.upgrade()
     
-    #if DEBUG
-      KeymanEngine.log.outputLevel = .debug
-      KeymanEngine.log.logAppDetails()
```

Comment:
Dropping it from the samples is one thing... but also from the OEM app we directly maintain?

Reply:
What information recorded by logAppDetails do you consider most useful? If there is anything that we really need that is not provided by the console for each log entry, then I could write a similar method for each logger.

Reply:
Huh, so that was actually an `XCGLogger` function - I didn't realize that.  For reference:

https://github.com/DaveWoodCom/XCGLogger/blob/a9c4667b247928a29bdd41be2ec2c8d304215a54/Sources/XCGLogger/XCGLogger.swift#L375-L410

Looking at what it actually would've been recording... yeah, we pretty much get all of it that's relevant via Sentry.  Looks like it is perfectly fine to eliminate, now that the details involved are clear.